container/root/etc/services-available/php-fpm/.command

```diff
@@ -6,7 +6,7 @@ set -o pipefail
 # IMPORTANT: PHP 7.3+ optionally allows undecorated stdout/stderr, removing pipe magic requirement
 UPGRADE_COMMAND=`grep "decorate_workers_output = no" $CONF_FPMPOOL`
 
-if [ $UPGRADE_COMMAND ]; then
+if [ -n "$UPGRADE_COMMAND" ]; then
```

Comment:
`-n string True if the length of string is non-zero` 😅

Reply:
@ingluisjimenez yeah - the subcommand is returning stdout, *not* the exit code